source/_components/media_player.denonavr.markdown

```diff
@@ -48,6 +48,8 @@ media_player:
     name: NAME
     show_all_sources: True / False
     timeout: POSITIVE INTEGER
+    sound_mode: True
```

Comment:
We are trying to keep the examples as minimal as possible. Please remove the added optional configuration paramters.

Reply:
I removed the sound_mode. But I would like to keep the sound_mode_dict in the example because this is a somewhat difficult parameter, for example if you put " " around it and make it a string it will not work anymore. And if I remember corectly the values in this dict schould be uppercase, otherwise the values received from the AVR will not match. And a dictionary is in general a little more difficult then just a string or a number for the basic users.

Furthermore the hole zones parameter is also in the example and you do not need that in general and is also optional if you don't use second zones on your AVR.